### DISCUSSION TITLE
Feature to embed external websites / webpages as iFrame.

### DISCUSSION THREAD

**Discussion by sksar** [directus/directus#17797] [Feature Requests]
### Summary

Hi,
It would be interesting to have a module using which we could embed external links directly inside of Directus as iFrames.
We could have external analytics tools, or BI tools, or possibly anything with an iFrame that we can embed.

It would be configurable from the project settings.

### Motivation

This is to add external embeds support right into the Directus dashboard.
This feature would provide useful to many looking to use or build other systems with the Database or the Directus API,
and embedding them directly in the Directus dashboard.

### Detailed Design

This can be approached in two ways, 

1. Module based: In this approach, the Modules bar settings already present in the project settings can be extended to support embeds. (Currently, it does support links), but it takes it to a new tab. That could be extended to allow embed.

2. Page based: In this approach, there would be a single module, maybe called external, and then inside of the external the Directus Navigation will be used to add as many links/embeds as the user would want.

### Must Have:
- Embedding URLs in iFrame

### Should Have:
- Feature to configure them from the settings page

### Could Have:
- Role-based configurations, so that different roles can have different options

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

Not a breaking change. Easy adoption.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by fitu86** [comment]
![InShot_20230630_105530236](https://github.com/directus/directus/assets/16139127/8feeee33-dccf-4dd0-a952-d5b7280d2acd)

It works with the url of the same project. Is it a matter of permissions to be able to preview external pages?

**Reply by paescuj** [reply]
The issue you're describing is probably this one: https://github.com/directus/directus/issues/18765
